io/disk/lvsetup.py

```diff
@@ -98,8 +102,8 @@ def setUp(self):
             else:
                 self.lv_size = int(self.lv_size) / 1024 / 1024
 
-        if self.disk:
-            disk_size = lv_utils.get_devices_total_space(self.disk.split())
+        if self.disks:
+            disk_size = lv_utils.get_devices_total_space(self.disks)
```

Comment:
self.disks can be more than one disk based on line #58.
so please take care of this, either you have to pass a single disk or run in a loop

Reply:
its already taken care as discussed offline